src/App/App.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import './App.css';
+import apikey from '../apikey';
+import Home from '../Home/Home';
+import Error from '../Error/Error';
+import Gallery from '../Gallery/Gallery';
+import Colors from '../Colors/Colors';
+import ArtInfo from '../ArtInfo/ArtInfo';
+
+import { fetchTodaysColor, getAllColors, getArt, getReplacement } from '../apiCalls';
+
+import { Switch, Route, Redirect } from 'react-router-dom';
+
+class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      today: new Date(),
+      isLoading: true,
+      todaysColor: {},
+      art: {},
+      colors: {},
+      currentColor: null,
+      error: null,
+    }
+  }
+
+  componentDidMount() {
+    fetchTodaysColor(this.getDayOfYear(), apikey)
+    .then(
+      (data) => this.setState({
+      todaysColor: data,
+      isLoading: false
+    },
+    (error) => {
+      console.error(error)
```

Comment:
There is an error coming from this line that is `undefined`, I suspect it is an error from the API but I can't figure it out. It's not causing any problems, it's just annoying.